using_openshift/executing_remote_commands.adoc

```diff
@@ -0,0 +1,76 @@
+= Executing Remote Commands
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+OpenShift takes advantage of a feature built into Kubernetes to support
+executing commands in containers. This is implemented using HTTP along with a
+multiplexed streaming protocol such as link:http://www.chromium.org/spdy[*SPDY*]
+or link:https://http2.github.io/[*HTTP/2*], and can be used to connect to a
+container via SSH as well.
```

Comment:
This makes it sound like SSH support is built in, which it isn't. You have to go through some hoops to get SSH working.